src/components/Navbar.tsx

```diff
@@ -0,0 +1,74 @@
+import Image from 'next/image'
+import ShoppingCart from '@mui/icons-material/ShoppingCartTwoTone'
+import Link from 'next/link'
+import AccountMenu from './AccountMenu'
+import { Badge, Skeleton } from '@mui/material'
+import { useEffect, useState } from 'react'
+import { URL } from '@/api/constans'
+import CategoriesTabs from './CategoriesTabs'
+
+const Navbar = () => {
+	const [isLoading, setIsLoading] = useState(true)
+	const [activeCategory, setActiveCategory] = useState<null | number>(0)
+	const [activeSubCategory, setActiveSubCategory] = useState<null | number>(null)
+
+	useEffect(() => {
+		const getPageContent = async () => {
```

Comment:
What is the purpose of this call?

Reply:
it was a delay in loading the categories styles because I was thinking that the font was flickering while loading but it was just btns without styles loaded yet so I added few inline styles to add priority to those styles